### PR TITLE
4.2.15

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by matthiasdiener** [comment]
ready for review @conda-forge/hdf4 @hmaarrfk

**Comment by hmaarrfk** [comment]
Looking at the maintainers of this repo, my guess is that they are pretty busy.

Do you want to add yourself as one? It would likely speed up the approval ;)

**Comment by matthiasdiener** [comment]
> Looking at the maintainers of this repo, my guess is that they are pretty busy.
> 
> Do you want to add yourself as one? It would likely speed up the approval ;)

Good idea, done =)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -61,3 +61,4 @@ extra:
     - groutr
     - msarahan
     - ocefpaf
+    - matthiasdiener
```
Comment (ocefpaf) [inline recipe/meta.yaml:64]: Thanks!

**Pull request merged**